frontend/src/views/main/admin/settings/Settings.vue

```diff
@@ -1,15 +1,18 @@
 <template>
-  <v-container fluid id="container-project-tab" class="vertical-container">
+  <div>
```

Comment:
there's still `#container-project-tab` style in Project.vue

we should either apply again or get rid of it

Reply:
Yes actually it's used in `project.vue`. I think it was inside `setting.vue` as an error since the id is not really related to the content itself